azure-pipelines.yml

```diff
@@ -49,7 +50,7 @@ jobs:
     variables:
       solution: 'WebJobs.Extensions.DurableTask.sln'
       buildPlatform: 'Any CPU'
-      buildConfiguration: 'Debug'
+      buildConfiguration: 'Release'
```

Comment:
Why is this change necessary?

Reply:
Release bits run faster, plus we fail release builds on warnings, so this will prevent warnings from slipping through PRs.